CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 #### Loki
 
 ##### Enhancements
+* [6821](https://github.com/grafana/loki/pull/6821) **kavirajk**: Introduce new cache type `embedded-cache` which is a in-process cache system that runs loki without any dependencies of external cache (like memcached, redis, etc). It can be run in two mode `distributed: false` (default and same as old `fifocache`) and `distributed: true` which runs cache is distributed fashing sharing keys between peers if Loki is run in microservices or SSD mode.
```

Comment:
```suggestion
* [6821](https://github.com/grafana/loki/pull/6821) **kavirajk**: Introduce new cache type `embedded-cache` which is an in-process cache system that runs loki the need for an external cache (like memcached, redis, etc). It can be run in two modes `distributed: false` (default, and same as old `fifocache`) and `distributed: true` which runs cache in distributed fashion sharding keys across peers if Loki is run in microservices or SSD mode.
```

Reply:
I think we should add a note in the upgrade guide as well, and mark `fifocache` as deprecated